extensions/azcli/src/api.ts

```diff
@@ -3,67 +3,39 @@
  *  Licensed under the Source EULA. See License.txt in the project root for license information.
  *--------------------------------------------------------------------------------------------*/
 
-import * as azdataExt from 'azdata-ext';
-import * as vscode from 'vscode';
-import { IAzdataTool, isEulaAccepted, MIN_AZDATA_VERSION, promptForEula } from './azdata';
+import * as azExt from 'az-ext';
+import { IAzTool } from './azdata';
 import Logger from './common/logger';
-import { NoAzdataError } from './common/utils';
-import * as constants from './constants';
+import { NoAzureCLIError } from './common/utils';
 import * as loc from './localizedConstants';
-import { AzdataToolService } from './services/azdataToolService';
+import { AzToolService } from './services/azToolService';
 
 /**
  * Validates that :
- *	- Azdata is installed
- *	- The Azdata version is >= the minimum required version
- *	- The Azdata CLI has been accepted
- * @param azdata The azdata tool to check
- * @param eulaAccepted Whether the Azdata CLI EULA has been accepted
+ *	- Az is installed
+ *	- The Az CLI has been accepted
```

Comment:
This is no longer being checked

Reply:
If I changed this function to validate that the correct version of Azure CLI Arc Extension is installed, would that be too much overhead to have to search the path and parse the output each time? Or do we want to skip the validate and just delete this function?

Reply:
I'd definitely be wary of doing that check every single time. I'd suggest leaving the logic as it is for now (only check once at startup) and then come back later once the initial work is done and investigating that option as a future improvement.

Reply:
The comment still says `The Az CLI has been accepted` - which first off isn't exactly clear what it's referring to and secondly assuming that it's referring to the EULA like the azdata one checked isn't actually being done. So I'd just remove this line.